src/routes.tsx

```diff
@@ -169,84 +169,53 @@ export const reactPageRoutes: ReactPageRoute[] = [
                   baseUrl: `/${locale}/new-member/home-accident`,
                   name: 'Web Onboarding DK - Danish Contents-Accident',
                 }
-              case 'home-accident-travel':
-                return {
-                  baseUrl: `/${locale}/new-member/home-accident-travel`,
-                  name: 'Web Onboarding DK - Danish Contents-Accident-Travel',
-                }
             }
             break
           case 'no':
-            switch (name) {
-              case 'contents':
-                return {
-                  baseUrl: '/no/new-member/contents',
-                  name: 'Web Onboarding NO - Norwegian Contents',
-                }
-              case 'travel':
-                return {
-                  baseUrl: '/no/new-member/travel',
-                  name: 'Web Onboarding NO - Norwegian Travel',
-                }
-              case 'combo':
-                return {
-                  baseUrl: '/no/new-member/combo',
-                  name: 'Web Onboarding NO - Norwegian Combo',
-                }
-            }
-            break
           case 'no-en':
             switch (name) {
               case 'contents':
                 return {
-                  baseUrl: '/no-en/new-member/contents',
-                  name: 'Web Onboarding NO - English Contents',
+                  baseUrl: `/${locale}/new-member/contents`,
+                  name: `Web Onboarding NO - ${
+                    locale === 'no' ? 'Norwegian Contents' : 'English Contents'
+                  }`,
```

Comment:
How about this thing? Do we want to keep the "English/Norwegian" as part of the name property. I saw that we did it slightly differently in DK.

What is it used for?

Reply:
Yes unfortunately we need to keep that! 😔  The reason is that when the Danish flows were created we had added an integration with Lokalise in order to handle different languages. Hence only one Embark/Angel flow per insurance product is needed.
But the older flows, the Norwegian and Swedish ones, don't have that. So there we have two flows per product - one in English and one in the local language. 
I hope that's understandable 😅 It's a bit complicated hehe..

Reply:
The `name` here is what we use as input to fetch the correct flow in `client/pages/Embark/index.tsx`
![image](https://user-images.githubusercontent.com/42962286/117135631-82c4a080-ada7-11eb-82c1-f10a5d1dcd25.png)

Reply:
Ah of course, that was me that mislead you @robinandeer since I was like "it's much cleaner for the DK routes, you can probably go ahead and refactor that" 😂

Reply:
haha okey thanks @styrelius! I made a quick update to bundle them in an enum to make it more obvious :)